src/Products/urban/migration/update_260.py

```diff
@@ -12,3 +13,15 @@ def add_couple_to_preliminary_notice(context):
     setup_tool.runImportStepFromProfile('profile-Products.urban:preinstall', 'typeinfo')
     setup_tool.runImportStepFromProfile('profile-Products.urban:preinstall', 'workflow')
     logger.info("upgrade step done!")
+
+
+def fix_inspection_title(context):
```

Comment:
This upgrade step should be removed, only future title will have the new format.